core/modules/qmeta/schema/migrate-3-to-4.sql

```diff
@@ -0,0 +1,15 @@
+--
+-- Migration script from version 3 to version 4 of QMeta database:
+--
+
+-- Increased storage capacity of the column `query` in the table QInfo
+ALTER TABLE `QInfo` MODIFY COLUMN `query` MEDIUMTEXT NOT NULL COMMENT 'Original query text as was submitted by client.';
+
+-- Increased storage capacity of the column `qTemplate` in the table QInfo
+ALTER TABLE QInfo MODIFY COLUMN `qTemplate` MEDIUMTEXT NOT NULL COMMENT 'Query template, string used to build final per-chunk query.';
+
+-- Increased storage capacity of the column `qMerge` in the table QInfo
+ALTER TABLE QInfo MODIFY COLUMN `qMerge` MEDIUMTEXT DEFAULT NULL COMMENT 'Merge (or aggregate) query to be executed on results table, result of this query is stored in merge table. If NULL then it is equivalent to SELECT *.';
+
+-- Increased storage capacity of the column `resultQuery` in the table QInfo
+ALTER TABLE QInfo MODIFY COLUMN `resultQuery` MEDIUMTEXT DEFAULT NULL COMMENT 'Query to be used by mysqlproxy to get final results.';
```

Comment:
`QInfo` is missing backticks in the last three `ALTER TABLE` statements. I think it would be better to make a single `ALTER TABLE` and include all four `MODIFY COLUMN` in it.